src/Blog/Entity/Comment.php

```diff
@@ -24,7 +24,7 @@
  *     }
  * )
  */
-class Comment
+final class Comment
```

Comment:
It is bad idea to do finalized entity classes when you are using cycle orm

Reply:
`Cycle/Orm` extends the identity classes, it's added in the docs, and I change it.

Reply:
fixed.